taxonomies/food/ingredients.txt

```diff
@@ -16722,24 +16707,14 @@ it: Olio di palmisti frazionato
 
 < en:palm kernel oil
 < en:palm oil
-en: palm and palm kernel oil
-de: Palm- und Palmkernöl, Palmöl und Palmkernöl
+en: palm and palm kernel oil, palm or palm kernel oil, palm oil or palm kernel oil, palm or palm kernel
+de: Palm- und Palmkernöl, Palmöl und Palmkernöl, Palmöl oder Palmkernöl, Palm- oder Palmkernöl
 es: aceite de palma y palmiste
-fi: palmu- ja palmuydinöljy
-fr: huile de palme et de palmiste
-hr: palma i ulje palminih koštica
-hu: pálma- és pálmamagolaj, pálmaolaj és pálmamagolaj
-nl: palmolie en palmkernolie
-
-< en:palm kernel oil
-< en:palm oil
-en: palm or palm kernel oil, palm oil or palm kernel oil, palm or palm kernel
-de: Palmöl oder Palmkernöl, Palm- oder Palmkernöl
-fi: palmu tai palmuydinöljy, palmuöljy tai palmuydinöljy, palmu tai palmuydin
-fr: huile de palme ou de palmiste, huile de palme ou huile de palmiste, palme ou palmiste
-hr: palmino ulje ili ulje palminih koštica
-hu: pálma- vagy pálmamagolaj, pálmaolaj vagy pálmamagolaj
-nl: palmolie of palmkernolie
+fi: palmu- ja palmuydinöljy, palmu tai palmuydinöljy, palmuöljy tai palmuydinöljy, palmu tai palmuydin
+fr: huile de palme et de palmiste, huile de palme ou de palmiste, huile de palme ou huile de palmiste, palme ou palmiste
+hr: palma i ulje palminih koštica, palmino ulje ili ulje palminih koštica
+hu: pálma- és pálmamagolaj, pálmaolaj és pálmamagolaj, pálma- vagy pálmamagolaj, pálmaolaj vagy pálmamagolaj
+nl: palmolie en palmkernolie, palmolie of palmkernolie
```

Comment:
@aleene @stephanegigandet 
Taxonomy editor merge those, because product opener will merge them, as "and" and "or" are stopwords.

Is it ok to keep it like this ?